src/Mvc/Mvc.Razor.RuntimeCompilation/src/Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation.csproj

```diff
@@ -12,7 +12,7 @@
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Mvc.Razor.Extensions" />
     <Reference Include="Microsoft.AspNetCore.Mvc.RazorPages" />
-    <Reference Include="Microsoft.AspNetCore.Razor.Runtime" />
+    <ProjectReference Include="..\..\..\Razor\Razor.Runtime\src\Microsoft.AspNetCore.Razor.Runtime.csproj" />
```

Comment:
I'd really prefer if we could avoid introducing a dependency between the compilation system and the runtime. I'll keep reading to see if this is really required.

Reply:
I couldn't build locally otherwise, the build seemed to use the nuget packages (the new interfaces I've added in AspNetCore.Razor weren't referencable in AspNetCore.Mvc.Razor)

Reply:
Nevermind, I see this was already here.

Reply:
I'm not sure why you're having a problem building locally, but I'm sure will figure that out by the end.